jetty-runtime/jetty9-base/pom.xml

```diff
@@ -122,7 +135,7 @@
           <arguments>
             <argument>-jar</argument>
             <argument>../jetty-distribution-${jetty.version}/start.jar</argument>
-            <argument>--add-to-startd=gae,http,jsp,jstl</argument>
+            <argument>--add-to-startd=http,deploy,jsp,jstl,http-forwarded,resources,gae</argument>
```

Comment:
@gregw  Can you clarify what each of these additions do?
deploy, http-forwarded, and resources